src/gt4py/next/common.py

```diff
@@ -18,9 +18,9 @@
 import dataclasses
 import enum
 import functools
-from collections.abc import Sequence, Set
-from typing import overload
-
+import sys
+from collections.abc import Sequence
+from typing import overload, Set, AbstractSet
```

Comment:
```suggestion
from collections.abc import Sequence, Set
from typing import overload
```
we want the metaclass such that UnitRange gets comparison operators and stuff